src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/CommonStepDef.scala

```diff
@@ -64,6 +64,14 @@ class CommonStepDef extends BaseStepDef {
     CommonPage.clickOnCheckbox(checkbox2Title)
   }
 
+  When("the user clicks all tailoring options")  { () =>
```

Comment:
While this works functionally, would it not make more sense to split this out per income source?